src/js/components/DataChart/stories/Prediction.js

```diff
@@ -0,0 +1,155 @@
+    background-color: #FFFFFF;
+  }
+`;
+const AnimatedBox = styled(Box)`
+  animation: ${keyFrameExampleOne} 3s linear;
+`;
+
+const data = [];
+for (let i = 0; i < 13; i += 1) {
+  const v = -Math.sin(i / 2.0);
+  const v2 = Math.cos(i / 2.0);
+  data.push({
+    date: `2020-07-${((i % 30) + 1).toString().padStart(2, 0)}`,
+    amount: Math.floor(v * 100),
+    need: Math.floor(v2 * 10),
+  });
+}
+
+for (let i = 0; i < 13; i += 1) {
+  const v = -Math.sin(i / 2.0);
+  const v2 = Math.cos(i / 2.0);
+  data.push({
+    date: `2020-08-${((i % 30) + 1).toString().padStart(2, 0)}`,
+    amountPredicted: Math.floor(v * 100),
+    needPredicted: Math.floor(v2 * 10),
+  });
+}
+
+const Example = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" justify="start" pad="large">
+      <Stack anchor="top-right" interactiveChild="first">
+        <DataChart
+          data={data}
+          series={[
+            'date',
+            {
+              property: 'amount',
+              // color: 'graph-3',
+              // Should the color prop have an impact? because it doesn't
+              point: 'circle',
+              label: 'Amount',
+            },
+            {
+              property: 'need',
+              color: 'graph-1', // does not have any impact
+              point: 'star', // does not have any impact
+              label: 'Demand',
+            },
+            {
+              property: 'amountPredicted',
+              // color: 'graph-2', // does not have any impact
+              point: 'circle',
+              label: 'Predicted Amount',
+            },
+            { property: 'needPredicted', color: 'graph-1' },
+          ]}
+          chart={[
+            {
+              property: 'amount',
+              type: 'area',
+              thickness: 'xsmall',
+              color: { color: 'graph-3', opacity: 'medium' },
+            },
+            {
+              property: 'amount',
+              type: 'line',
+              thickness: 'xsmall',
+              round: true,
+            },
+            {
+              property: 'amountPredicted',
+              type: 'area',
+              thickness: 'xsmall',
+              color: { color: 'graph-3', opacity: 'medium' },
+            },
+            {
+              property: 'amountPredicted',
+              type: 'line',
+              thickness: 'xsmall',
+              round: true,
+              dash: true,
+            },
+            {
+              property: 'amountPredicted',
+              type: 'point',
+              thickness: 'small',
+              point: 'circle', // shouldn't be here
+            },
+            {
+              property: 'amount',
+              type: 'point',
+              thickness: 'small',
+            },
+            // {
+            //   property: 'needPredicted',
+            //   type: 'line',
+            //   thickness: 'xxsmall',
+            //   round: true,
+            //   dash: true,
+            // },
```

Comment:
Uncommenting this block causes the layout to break.

Reply:
That's because the Legend starts driving the width. And, it's not breaking the layout of the DataChart, just the way the Stack is being used in the story. I think we should either:
- add animation to DataChart
- add the ability to insert your own elements within the Stack inside DataChart
But, I think we can add those later on.